msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -1879,7 +1879,7 @@ public void onCancel() {
         };
     }
 
-    private DeviceCodeFlowCommandCallback getDeviceCodeFlowCommandCallback(@NonNull final DeviceCodeFlowCallback callback) {
+    protected DeviceCodeFlowCommandCallback getDeviceCodeFlowCommandCallback(@NonNull final DeviceCodeFlowCallback callback) {
```

Comment:
This is going to expose this as a method on instances of PCA - we probably don't want this. I mean do our customers need to know about command callbacks? Probably not

Reply:
I mean it is fine because the customers would have to create a sub-class of PCA or try to create something in the same package..but I was just wondering if there was a way to avoid this

Reply:
There may be a smarter way to do this, I essentially only needed to expose the method to SingleAccountPCA, so private wouldn't work. This change would make it the same visibility as the base getCommandCallback() method in PCA.

Reply:
Yes, it's fine.